doc/source/reference/functions/time_classify_day.rst

```diff
@@ -8,11 +8,111 @@
 Summary
 -------
 
+``time_classify_day`` rounds date to a day unit.
```

Comment:
"date" -> "time"

Reply:
Thank you for your comment.
I've fixed it.